azkaban-exec-server/src/test/java/azkaban/dag/NullFlowProcessor.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.dag;
+
+public class NullFlowProcessor implements FlowProcessor {
+
+  @Override
+  public void changeStatus(final Flow flow, final Status status) {
+    return;
```

Comment:
Wouldn't a default Mockito mock do exactly the same?

Reply:
Good point.

Reply:
done